package.json

```diff
@@ -65,6 +66,11 @@
     "@modelcontextprotocol/sdk": ">=1.12.0 <=1.12.1",
```

Comment:
This seems too strict. You typically want to make a peer dependency work with all major versions since thats the only time its breaking.

Something like 

```
"@modelcontextprotocol/sdk": "^1.12.0",
```